lab1.js

```diff
@@ -105,6 +127,13 @@ var nextAnimal;
 // Assign one of your favorite animals to nextAnimal using Math.random() to pick
 
 // your code goes here
+function pickNextAnimal(favoriteAnimals) {
+  var a = Math.floor(Math.random() * 4);
+  nextAnimal = favoriteAnimals[a];
+  return nextAnimal;
+}
+
+pickNextAnimal(favoriteAnimals);
```

Comment:
here, this works because you assign the new value to nextAnimal in the function (the return doesn't really do anything in this instance)